Library/Formula/elixirscript.rb

```diff
@@ -0,0 +1,20 @@
+class Elixirscript < Formula
+  desc "Elixir to JavaScript compiler"
+  homepage "https://github.com/bryanjos/elixirscript"
+  url "https://github.com/bryanjos/elixirscript/archive/v0.16.0.tar.gz"
+  sha256 "1ccc52501be1d762e823c6174aa5db1de200f4edc96b39f1057fa8aefeb6212e"
+
+  depends_on "elixir" => :build
+
+  def install
+    system "mix", "local.hex", "--force"
+    system "mix", "deps.get"
+    system "mix", "escript.build"
+    bin.install "elixirscript"
+    prefix.install "priv/Elixir.js", "LICENSE"
+  end
+
+  test do
+    system "elixirscript", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!